packages/smooth_app/lib/pages/product/confirm_and_upload_picture.dart

```diff
@@ -52,6 +52,8 @@ class _ConfirmAndUploadPictureState extends State<ConfirmAndUploadPicture> {
     return SmoothScaffold(
       backgroundColor: Colors.black,
       appBar: AppBar(
```

Comment:
Could you switch to a SmoothAppBar, to have this Widget used on many places as possible in the app?
(it won't change anything for you)

Reply:
@g123k Sure!